README.md

```
@@ -29,8 +26,8 @@ aether is free and open-source software. To host this egg, alot of resources and
 4. ???
 5. profit
 
-## 🖥️ Hostings that use aether
-| Hostings                                    | About                                                                                                                                        | Custom banner? | Notes    |
+## 🖥️ Hosting's that use aether
+| Hosting's                                    | About                                                                                                                                        | Custom banner? | Notes    |
```

markdownlint-cli2 (0.18.1)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)